### PR TITLE
Initial creation of kubectl plugin for fdb

### PR DESCRIPTION
**Pull request by johscheuer**
Initial support for a Kubectl plugin to interact with the fdb operator. This add the skeleton for a Kubectl fdb plugin to simplify working with the operator and fdb clusters on Kubernetes. Further commands can be added afterwards.

Solves: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/131

### CODE REVIEW

File: kubectl-fdb/cmd/exclude.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"log"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ctx "context"
+	fdbtypes "github.com/FoundationDB/fdb-kubernetes-operator/api/v1beta1"
+)
+
+// excludeCmd represents the exclude command of the fdb cli
+var excludeCmd = &cobra.Command{
```
Comment (johscheuer) [inline kubectl-fdb/cmd/exclude.go:35]: In an additional PR we could also add the following features:

- exclude all instances of a cluster
- exclude all instances with a specific label (e.g. all storage instances)

I will open an follow up issue for this task.

File: kubectl-fdb/Readme.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+export PATH="$PATH:$(pwd)/bin" 
+```
+
+## Usage
+
+Run `kubectl fdb help` to get the latest help.
+
+### Supported operations
+
+- `exlcude` instances from a cluster.
```
Comment (brownleej) [inline kubectl-fdb/Readme.md:21]:
```suggestion
- `exclude` instances from a cluster.
```

File: kubectl-fdb/cmd/exclude.go (hunk 1 above)
Comment (brownleej) [reply kubectl-fdb/cmd/exclude.go:37]: Wouldn't those operations fail due to excluding too much of the cluster?

Comment (brownleej) [reply kubectl-fdb/cmd/exclude.go:37]: Ah, I was confused about what the `exclude` operation does. I think we need to be consistent in our use of `exclude` and `replace`. We may want to talk about this more out-of-band to get the right terminology.

File: kubectl-fdb/cmd/version.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+)
+
+// TODO: should follow the operator versioning
+var pluginVersion string = "0.0.1"
```
Comment (brownleej) [inline kubectl-fdb/cmd/version.go:33]: Should we make this match the current version, and then add a note to the release guide to update the version here? We're already updating the version by find-and-replace.

**Review by brownleej — CHANGES_REQUESTED** [review]

File: kubectl-fdb/cmd/exclude.go (hunk 1 above)
Comment (johscheuer) [reply kubectl-fdb/cmd/exclude.go]: I adjusted the name to `remove` like discussed "offline"

File: kubectl-fdb/cmd/remove.go [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ctx "context"
+
+	fdbtypes "github.com/FoundationDB/fdb-kubernetes-operator/api/v1beta1"
+)
+
+// removeCmd represents the removal of
```
Comment (brownleej) [inline kubectl-fdb/cmd/remove.go:39]: Is there meant to be more in this comment?

**Review by brownleej — CHANGES_REQUESTED** [review]
Small question about a comment. Otherwise this looks good.

File: kubectl-fdb/cmd/remove.go (hunk 4 above)
Comment (johscheuer) [reply kubectl-fdb/cmd/remove.go]: I fixed it, I probably missed this one during refactoring.

**Review by brownleej — APPROVED** [review]
👍

**Pull request merged**